pandas_gbq/load.py

```diff
@@ -65,6 +66,10 @@ def load_chunks(
 
     if schema is None:
         schema = pandas_gbq.schema.generate_bq_schema(dataframe)
+    else:
+        # See Issue #277
+        # Prevent inplace modification of toplevel function argument
+        schema = copy.deepcopy(schema)
```

Comment:
Might make sense to do this at the `to_gbq` level instead, since that's the main public entrypoint.

Reply:
I see what you're saying, but I would argue at the same time you'd want to cure things at the root.  If you have some other library that's calling `load_chunks()` (I guess it's technically public API?) then it may be subject to the same problem even if fixed in `to_gbq()`.